Person.cs

```diff
@@ -0,0 +1,47 @@
+﻿using PhoneBook.StaticHelpers;
+using System.Collections.Generic;
+
+namespace PhoneBook
+{
+    public class Person
+    {
+        private const int rowCountWithSurname = 4;
+        private const int rowCountWithoutSurname = 3;
+        public static List<Person> TextToPersonInfo(string text, List<string> separatorErrors)
+        {
+            bool isValidSeporator;
+            List<Person> people = new List<Person>();
+            string[] rows = text.TextToRowSpliter();
+            for (int i = 0; i < rows.Length; i++)
+            {
+                isValidSeporator = true;
+                string[] substring = rows[i].RowToWordSpliter();
+                if (substring.Length == rowCountWithSurname)
+                {
+                    people.Add(new Person(name: substring[0], number: substring[3], surname: substring[1]));
```

Comment:
Create variables for name, surname and number and use them to make sure your code is more readable.
Having `name` variable is more readable than `substring[0]`

Reply:
There are variables in there and i am giving them values depending on the format from the substring

Reply:
I mean you can create new variables like: 
`string name = substring[0];`
`string name = substring[0];`